mitiq/pec/representations/learning.py

```diff
@@ -84,3 +98,78 @@ def _biased_noise_loss_function(
     return np.sum((mitigated_values - ideal_values) ** 2) / len(
         training_circuits
     )
+
+
+def _unmitigated_loss_function(
+    epsilon_guess: np.ndarray,
+    noisy_executor: Executor,
+    training_circuits: List[QPROGRAM],
+    observable: Optional[Observable] = None,
+) -> float:
+    def ideal_execute(circ: Circuit) -> np.ndarray:
+        return compute_density_matrix(circ, noise_level=(0.0,))
+
+    def noise_model_execute(circ: Circuit) -> np.ndarray:
+        circuit = convert_to_mitiq(circ)[0]
+        noisy_circ = circuit.with_noise(
+            biased_noise_channel(epsilon=epsilon_guess[0], eta=0)
+        )
+        return ideal_execute(noisy_circ)
+
+    noise_model_executor = Executor(noise_model_execute)
+    noise_model_values = np.array(
+        [
+            noise_model_executor.evaluate(training_circuit, observable)
+            for training_circuit in training_circuits
+        ]
+    )
+
+    noisy_values = np.array(
+        [
+            noisy_executor.evaluate(training_circuit, observable)
+            for training_circuit in training_circuits
+        ]
+    )
+
+    return np.sum(
+        abs(noise_model_values.reshape(-1, 1) - noisy_values.reshape(-1, 1))
+        ** 2
+    ) / len(training_circuits)
```

Comment:
I believe the `reshape` calls are unneeded here. The return value can be simplified as well if you prefer this:
```py
return np.linalg.norm(noise_model_values - noisy_values) ** 2 / len(training_circuits)
```